src/elin_bjoernson_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+
+def bubble_sort(numbers):
+    new_list = list(numbers)
+    for _ in new_list:
+        for idx, num in enumerate(new_list):
```

Comment:
Please check the algorithm. There will be redundant comparison when you compare already sorted elements.